Makefile

```diff
@@ -81,16 +81,17 @@ endif
 
 integration-tests:
 	$(CARGO_BINARY) test --features "integration_tests" integration_tests::
+	$(CARGO_BINARY) test --all-features
 
 regression-tests:
 	chmod +x end-to-end-tests/ci/chunked-upload.sh
-	./end-to-end-tests/ci/chunked-upload.sh
+	end-to-end-tests/ci/chunked-upload.sh
 	chmod +x end-to-end-tests/ci/direct-execution.sh
```

Comment:
`direct-exectuion.sh` and the other two scripts are empty.
They should be removed from the repo and from the Makefile.

Reply:
It was just mode changes between `644` and `755`, I removed the changes from the PR.

Reply:
Yeah, I was just asking in general.

Maybe create a follow-up ticket to clean up the empty tests, because it's confusing to have them in there.

Reply:
Which empty tests? They aren't empty? GitHub just shows "empty file" because that's what git commits when a mode changes.